src/main/docker/base/Dockerfile

```diff
@@ -1,3 +1,17 @@
+# Copyright (C) 2019 Knot.x Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 # A base Dockerfile for Knot.x
 
 FROM openjdk:8-jre
```

Comment:
LABEL maintainer="Knot.x Project"

Reply:
It's there, just GitHub is not presenting the full file:
https://github.com/Knotx/knotx-docker/blob/feature/release-automation/src/main/docker/base/Dockerfile#L18